fabtests/common/shared.c

```diff
@@ -2907,6 +2907,44 @@ int ft_get_tx_comp(uint64_t total)
 	return ret;
 }
 
+static inline int ft_check_for_unexpected_comps()
```

Comment:
The motivation for this change is detecting whether we really have a duplicate completion, this is for experimental purpose so I do not think we want this detection in the production code.

Reply:
Ok, will delete it